dashboards/google-vertex-ai/vertex-ai-custom-training.json

```diff
@@ -0,0 +1,373 @@
+{
+  "displayName": "Vertex AI Custom Training Dashboard",
+  "dashboardFilters": [
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "job_id",
+      "stringValue": "",
+      "templateVariable": "",
+      "valueType": "STRING"
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "region",
+      "stringValue": "",
+      "templateVariable": "",
+      "valueType": "STRING"
+    }
+  ],
+  "labels": {},
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "height": 4,
+        "width": 48,
+        "widget": {
+          "title": "CPU and Memory",
+          "id": "",
+          "sectionHeader": {
```

Comment:
Either section header or collapsible group is fine for organization, but let's just use one or the other (this is more standard in other dashboards)